config/prod/xschildw-packer-shinyserver-pro-check.yaml

```diff
@@ -1,10 +1,10 @@
 # Provision an EC2 instance
 template_path: remote/managed-ec2-linux-v2.j2
-stack_name: xschildw-shinyserver-pro
+stack_name: xschildw-shinyserver-pro-check
 sceptre_user_data:
   Distro: "ubuntu"     # (valid values: aws, ubuntu) AMIId must match distro
   # (Optional) Expose ports to incoming traffic (default is no open ports) (valid range: 1-65535)
-  OpenPorts: ["80","8080"]
+  OpenPorts: ["3838","4151"]
```

Comment:
remember, you don't need to set this if you connect to the VPN.  all ports are open for VPN

Reply:
Will do, I would need them open to allow a load balancer to come in. Correct?